CircleciScripts/prepare_xcframework_archives.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+import shutil
+import re
+
+from framework_list import xcframeworks
+from functions import log, run_command
+
+def create_archives(xcframework_path, archive_path, version):
+    os.makedirs(archive_path, exist_ok=True)
+    for framework in xcframeworks:
+
+        finalZipName = f"{framework}-{version}"
+        log(f"Creating zip file for {finalZipName}")
+
+        xcframework_sdk_path = f"{xcframework_path}/{framework}.xcframework"
```

Comment:
Here and throughout: prefer [`os.path.join`](https://docs.python.org/3/library/os.path.html#os.path.join) to construct paths rather than hardcoding separators

Reply:
Fixed